src/Common/Tooltip/constants.tsx

```diff
@@ -0,0 +1,3 @@
+export const TOOLTIP_CONTENTS = {
```

Comment:
Ideally should reside in fe-common

Reply:
I had thought of that but since they are **tooltip** contents they are more suited here. Anyways when these are exported it matters not wherein from fe-common-lib they are exported.